Dfc.CourseDirectory/Resources/AzureSearch/SearchIndex.json

```diff
@@ -22,5 +22,51 @@
 			{ "name": "Country", "type": "Edm.String", "searchable": false, "filterable": false, "sortable": false, "facetable": false },
 			{ "name": "updated", "type": "Edm.DateTimeOffset", "filterable": false, "sortable": false, "facetable": false }
 		]
+	},
+	{
+		"name": "provider",
+		"fields": [
```

Comment:
I thought we had agreed to leave the index schema largely the same? This appears to have changed considerably.

Reply:
I thought some more about the compatibility option when I got into the detail of it. When I understood the shape of the problem better I decided to make it a breaking change.

The existing names are misaligned and confusing, there are mappings between very similarly named properties as you go up the layers. There are also some unused fields for which I didn't see an obvious replacement source of data.

As I understand it the only dependency on the existing index is the provider search UI for admin users. We have a single model for the search results which gives us a single place to update when we switch to the new index:
https://github.com/SkillsFundingAgency/dfc-coursedirectory/blob/7f0e9f194c0670bcc82f32afa98e2a3d17f6aac2/src/Dfc.CourseDirectory.Core/Search/Models/Provider.cs#L6

I think the switch to a replacement index is a convenient moment to eliminate the arbitrary variation in naming, unused fields, and confusing mappings.